lua/nvim-tree/live-filter.lua

```diff
@@ -104,25 +104,38 @@ local function configure_buffer_overlay()
   vim.api.nvim_buf_set_keymap(overlay_bufnr, "i", "<CR>", "<cmd>stopinsert<CR>", {})
 end
 
-local function create_overlay()
+local function calculate_overlay_win_width()
+  local wininfo = vim.fn.getwininfo(view.get_winnr())[1]
```

Comment:
`getwininfo` is hardened against nil winid.

I'm not sure how we'd get a nil however thank you for checking.

Simly returning 20 would be enough; subtracting 2 doesn't really do much.

Reply:
By doc it could be `nil`, even if  it happens we can secure by simple return 20. I don't think it reduces the readability of the code.

Reply:
> By doc it could be nil, maybe 1 time in 1 million and if it happens we can secure by simply return 20. I don't think it reduces the readability of the code.

Yup, that's the safest and most resilient solution.